packages/integration-sdk-cli/src/commands/generate-integration-graph-schema.ts

```diff
@@ -0,0 +1,226 @@
+import {
+  IntegrationInstanceConfig,
+  IntegrationStepExecutionContext,
+  Step,
+  StepEntityMetadata,
+  StepMappedRelationshipMetadata,
+  StepRelationshipMetadata,
+} from '@jupiterone/integration-sdk-core';
+import { createCommand } from 'commander';
+import path from 'path';
+import { loadConfig } from '../config';
+import { promises as fs } from 'fs';
+import * as log from '../log';
+
+export function generateIntegrationGraphSchemaCommand() {
+  return createCommand('generate-integration-graph-schema')
+    .description(
+      'generate integration graph metadata summary from step metadata',
+    )
+    .option(
+      '-o, --output-file <path>',
+      'project relative path to generated integration graph schema file',
+    )
+    .option(
+      '-p, --project-path <directory>',
+      'path to integration project directory',
+      process.cwd(),
+    )
+    .action(async (options) => {
+      const { projectPath, outputFile } = options;
+
+      log.info(
+        `Generating integration graph schema (projectPath=${projectPath}, outputFile=${outputFile})`,
+      );
+      const config = await loadConfig(path.join(projectPath, 'src'));
+
+      const integrationGraphSchema = generateIntegrationGraphSchema(
+        config.integrationSteps,
+      );
+
+      if (outputFile) {
+        await fs.writeFile(outputFile, JSON.stringify(integrationGraphSchema), {
+          encoding: 'utf-8',
+        });
+      } else {
+        console.log(JSON.stringify(integrationGraphSchema, null, 2));
```

Comment:
Do we want to document anywhere that the default action when no output file is specified is to print to stdout?